doc/bootloader-building.rst

```diff
@@ -71,26 +71,14 @@ You can run the following to install everything required:
 
 * On Debian- or Ubuntu-like systems::
 
-    sudo apt-get install build-essential
+    sudo apt-get install build-essential zlib1g-dev
 
 * On Fedora, RedHat and derivates::
 
     sudo yum groupinstall "Development Tools"
-
-* For other Distributions please consult the distributions documentation.
-
-If your distribution doesn't already have it you will also have to
-install ``zlib-devel``.
-
-* On Debian- or Ubuntu-like systems::
-
-    sudo apt-get install zlib1g-dev
-
-* On Fedora, RedHat and derivates::
-
     sudo yum install zlib-devel
```

Comment:
Same for this one, of course :-)

Reply:
Can you mix `groupinstall` and regular `install`\ s with `yum`?

Reply:
I don't think so. You can probably hack something using `yum shell`, to get all the packages in a single transaction.

Reply:
OK, in that case @htgoebel do we not have to leave this one as it is?

Reply:
This is fine.